### PR TITLE
Docs (backend): Updated Contributing.md

### LINKED ISSUE #42: [BUG]: Setting up backend
### Describe the bug

The backend is not initialised after following the steps as mentioned in readme which are as follows.
<br>
`docker compose up`
`cd backend`
`npm install`
`npm run start:dev`

### Steps To Reproduce

_No response_

### Expected Behavior

Backend should be up and running.

### Screenshots/Videos

_No response_

### What device are you using?

Desktop

### Operating System

Windows, Linux

### What browsers are you seeing the problem on?

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by NightFury742**
## Overview

1. This PR fixes  #42.
2. This PR does the following: Updated contributing.md to incorporate for the steps to run the backend server.

## Essential Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ ] The PR is made from a branch that's **not** called "main/master".

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -29,13 +29,16 @@ If you're reading this, you're probably creating a Pull Request or planning to d
    [Install](https://docs.docker.com/engine/install/)
```
Comment (prajjwalkapoor) [inline Contributing.md:30]: Write here:
After installing docker, run the following command to start database containers.

File: Contributing.md [hunk 2]
```diff
@@ -29,13 +29,16 @@ If you're reading this, you're probably creating a Pull Request or planning to d
...
    ```bash
-   docker compose up
+   docker compose --env-file backend/.env up
    ```
 
 5. In new terminal type
 
    ```bash
    cd backend
+   npx prisma generate
```
Comment (prajjwalkapoor) [inline Contributing.md:39]: We don't need Prisma generate command, the migrate command will automatically generate the Prisma client so you can remove this.

File: Contributing.md [hunk 3]
```diff
@@ -29,13 +29,16 @@ If you're reading this, you're probably creating a Pull Request or planning to d
...
+   npm run migrate:dev
+   npm run seed
    npm install
```
Comment (prajjwalkapoor) [inline Contributing.md:42]: It should be the first step after moving to the backend directory so move this up.

**Review by prajjwalkapoor — CHANGES_REQUESTED** [review]

**Comment by NightFury742** [comment]
@prajjwalkapoor I have made the requested changes, please review them.

**Review by prajjwalkapoor — APPROVED** [review]
LGTM.

**Pull request merged**